### PR TITLE
[TEMP PR] Release fixes

### PR DESCRIPTION
**Pull request by hadia206**
Making PR with my changes to monitor changes easily.
Once everything is work will open a different PR

### CODE REVIEW

File: .github/workflows/_build_bodo_conda_linux_comm.yml [hunk 1]
```diff
@@ -53,6 +53,7 @@ jobs:
           # Conda-Build normally uses a random path. Setting these args disables that.
           # --croot for consistent base directory path for sccache
           # --build-id-pat for consistent build directory path for sccache
+          # Why `PLATFORM_CHANNEL` here?
```
Comment (srilman) [inline .github/workflows/_build_bodo_conda_linux_comm.yml:56]: Yeah you're right, this is an old use, we don't need. Can you remove the use in the next line as well?

File: CMakeLists.txt [hunk 2]
```diff
@@ -22,9 +22,11 @@ find_program(CCACHE_EXECUTABLE "ccache")
 find_program(SCCACHE_EXECUTABLE "sccache")
 
 if ((NOT DEFINED ENV{DISABLE_CCACHE}) AND CCACHE_EXECUTABLE)
-  set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
+  set(CMAKE_CXX_COMPILER_LAUNCHER ccache)
```
[the remark this answers is not in the document]
Comment (srilman) [reply CMakeLists.txt:25]: Can revert now?

File: buildscripts/bodo/publish_binary_linux.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -exo pipefail
```
[the remark this answers is not in the document]
Comment (srilman) [reply buildscripts/bodo/publish_binary_linux.sh:2]: I'm not sure if those lines are even necessary, doesn't micromamba auto activate on the shell scripts always?

[the remark this answers is not in the document]
Comment (hadia206) [reply buildscripts/bodo/publish_binary_linux.sh:2]: You added them in this [PR](https://github.com/bodo-ai/Bodo-Archive/pull/8666/files#diff-12c6d1e61ad1fcdc918834c261fa1ce372c49f20a5362e6503d6bb2f70da512b)

**Comment by hadia206** [comment]
Opened actual PR [here](https://github.com/bodo-ai/Bodo/pull/44)

**Pull request closed without merging**